jenkinsbuilder/weekly-scan.py

```diff
@@ -111,7 +116,15 @@
             job = bs.put(json.dumps(data))
 
             # Initializing Database entries
-            project, created = Project.objects.get_or_create(name=project_name)
+            project, created = Project.objects.get_or_create(
+                name=project_name,
+                target_file_link=form_targetfile_link(
+                    git_url,
```

Comment:
why add an extra variable? Just use `entry["git-url"]`

same applies to following 3 more variables used.

Reply:
That is in keeping with rest of the pack. In that sense, there is no need for appid, jobid and desired tag either. It simply reduces the possibility of errors as intelli sense, and for that matter, compiler, can handle variable errors, but not mismatched strings.

Reply:
Well, the inefficient practices followed in the pack do not really need to be followed for the sake of similarity, while I don't expect the other part of the code to be also fixed with this PR. To comment on the compiler and variable errors, well, the tests should be there to take care of it.

You can keep it as is.